react-native-libraries.json

```
@@ -10466,5 +10466,15 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/powersync-ja/powersync-react-native-sdk",
```

Hello @DominicGBauer, thanks for adding a new entry to the directory! 👍 

Just one nit, since the repository is a monorepo, we need to use the full path to the place where selected package `package.json` resides.

```suggestion
    "githubUrl": "https://github.com/powersync-ja/powersync-react-native-sdk/tree/main/packages/powersync-sdk-react-native",
```